<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LightTaskManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="actionsGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Поиск</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="filterByTagLabel">
         <property name="text">
          <string>Тэг</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterByTagLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="filterByUserLabel">
         <property name="text">
          <string>Исполнитель</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filterByUserLineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="acceptFiltersPushButton">
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QFrame" name="filePathFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="filePathLabel">
         <property name="text">
          <string>Путь до файла:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filePathLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="taskContainerFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="taskContainerLabel">
         <property name="text">
          <string>Задачи</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QWidget" name="tasksContainerWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="tasksContainerHorizontalLayout">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="statusesLabelsHorizontalLayout"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QFrame" name="commandLineFrame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="commandLineLabel">
         <property name="text">
          <string>Команда:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="commandLineLineEdit"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="archiveByStatusGroupBox">
      <property name="title">
       <string>Архивация по статусу</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QLineEdit" name="archiveByStatusLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="archiveByStatusPushButton">
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionInitializeRepository"/>
    <addaction name="actionOpenRepository"/>
    <addaction name="actionAddTask"/>
    <addaction name="actionDeleteTask"/>
   </widget>
   <widget class="QMenu" name="menuEnvironment">
    <property name="title">
     <string>Environment</string>
    </property>
    <addaction name="actionSettings"/>
    <addaction name="actionOpenTerminal"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuEnvironment"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="currentTaskDockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Описание задачи</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <widget class="QFrame" name="taskInformationFrame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="currentTaskIndexLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QLineEdit" name="currentTaskDateLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="addTagToolButton">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/images/Plus.png</normaloff>:/images/images/Plus.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QLineEdit" name="currentTaskTitleLineEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="removeTagToolButton">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/images/Minus.png</normaloff>:/images/images/Minus.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="currentTaskTagsLabel">
          <property name="text">
           <string>Тэги</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="addUserToolButton">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>+</string>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/images/Plus.png</normaloff>:/images/images/Plus.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="1" rowspan="2">
         <widget class="QListWidget" name="currentTaskTagsListWidget"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="currentTaskTitleLabel">
          <property name="text">
           <string>Заголовок</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QToolButton" name="removeUserToolButton">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="text">
           <string>-</string>
          </property>
          <property name="icon">
           <iconset resource="resources/resources.qrc">
            <normaloff>:/images/images/Minus.png</normaloff>:/images/images/Minus.png</iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="currentTaskUsersLabel">
          <property name="text">
           <string>Пользователи</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1" rowspan="2">
         <widget class="QListWidget" name="currentTaskUsersListWidget"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="currentTaskDateLabel">
          <property name="text">
           <string>Дедлайн</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="currentTaskIndexLabel">
          <property name="text">
           <string>Индекс</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QCalendarWidget" name="calendarWidget"/>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="editTaskPushButton">
       <property name="text">
        <string>Редактировать</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="currentTaskDescriptionPlainTextEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="saveTaskPushButton">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionInitializeRepository">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/FileNew.png</normaloff>:/images/images/FileNew.png</iconset>
   </property>
   <property name="text">
    <string>Initialize Repository</string>
   </property>
  </action>
  <action name="actionAddTask">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/Plus.png</normaloff>:/images/images/Plus.png</iconset>
   </property>
   <property name="text">
    <string>Add Task</string>
   </property>
  </action>
  <action name="actionDeleteTask">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/Minus.png</normaloff>:/images/images/Minus.png</iconset>
   </property>
   <property name="text">
    <string>Delete Task</string>
   </property>
  </action>
  <action name="actionOpenRepository">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/FileOpen.png</normaloff>:/images/images/FileOpen.png</iconset>
   </property>
   <property name="text">
    <string>Open Repository</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resources/resources.qrc">
     <normaloff>:/images/images/Settings.png</normaloff>:/images/images/Settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionOpenTerminal">
   <property name="text">
    <string>Open Terminal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
